release-notes.md

```diff
@@ -1,5 +1,15 @@
 # Release notes
 
+## August 6, 2019
+
+The following updates were made in this release:
+
+**iOS Target 2.1.2 and Android Target 1.1.2**
+
+* New Configuration Boolean key `target.previewEnabled` provided which can be used to enable/disable Target Preview. If key is not provided, then preview will be enabled by default.
```

Comment:
Please reword the 1st bullet point as follows:

- A new  `target.previewEnabled`, a new configuration Boolean key has been added, and this key is used to enable/disable Target Preview. 
If key is not provided, preview is enabled by default. --> move to a new line under the bullet.

Reply:
changed to `target.previewEnabled`, a new configuration Boolean key, has been added. This key is used to enable/disable Target Preview.